features/generate.feature

```diff
@@ -53,3 +53,11 @@ Feature: Generate a new project using Tuist
     Then I should be able to test the scheme ATests
     Then I should be able to build the scheme B
     Then I should be able to test the scheme BTests
+
+Scenario: The project is an iOS application with a target dependency and transitive framework dependency (ios_app_with_transitive_framework)
+    Given that tuist is available
+    And I have a working directory
+    Then I copy the fixture ios_app_with_transitive_framework into the working directory
+    Then tuist generates the project
+    Then I should be able to build the scheme App
```

Comment:
I'd add an extra step 😬 
```ruby
Then the framework Framework1 has been copied into the product.
```
You can look up the app in the DerivedData directory and check if the framework exists in the `Frameworks` directory.

Reply:
I'd even extend it with:

```ruby
Then the frameworks Framework1, Framework2 has been copied into the product App.app and only contain architecture x86
```